apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -81,20 +82,17 @@ const PaymentsSumInfo: React.FC<IPaymentsSumInfoProps> = ({
     )
 }
 
-function getSumPerPeriod (objs) {
-    let sumAllPayments = 0,
-        sumDonePayments = 0,
-        sumWithdrawnPayments = 0
-
-    for (const key in objs) {
-        sumAllPayments += Number(objs[key].amount)
-        if (objs[key].status === PAYMENT_DONE_STATUS)
-            sumDonePayments += Number(objs[key].amount)
-        else
-            sumWithdrawnPayments += Number(objs[key].amount)
-    }
-
-    return { sumAllPayments, sumDonePayments, sumWithdrawnPayments }
+function GetSum (whereQuery, paymentStatus) {
```

Comment:
why function name starts with uppercase?

Reply:
Linter checks. Because of useQuery, it requires uppercase

Reply:
I have not seen it in our project, is this correct?